build_runner/test/generate/build_test.dart

```diff
@@ -226,7 +226,7 @@ void main() {
           throwsA(anything));
     });
 
-    group('with `writeToCache: true`', () {
+    group('with `hideOutput: true`', () {
```

Comment:
Probably worth adding an additional test to this file that mixes hideOutput settings for different build actions

Reply:
Done